From: Carlos Maddela <e7appew@gmail.com>
Date: Wed, 9 Aug 2017 20:49:31 +1000
Subject: Do not suppress the display of compilation commands

Description: Do not suppress the display of compilation commands,
 so that build logs may be analysed by blhc.
Author: Carlos Maddela <e7appew@gmail.com>
Forwarded: not-needed
Last-Update: 2017-08-9
---
diff --git a/GNUmakefile.template b/GNUmakefile.template
index 6191d9a..4eb0103 100644
--- a/GNUmakefile.template
+++ b/GNUmakefile.template
@@ -112,45 +112,45 @@ time-stamp:
 
 ${BUILDTMP}/closure.o: closure.c build.h
 	@echo "Compiling:" closure.c
-	@$(CC) $(COPTS) -c closure.c -o ${BUILDTMP}/closure.o
+	$(CC) $(COPTS) -c closure.c -o ${BUILDTMP}/closure.o
 
 dvdisaster: inlined-icons.h $(OFILES) 
 	@echo "Linking  : dvdisaster"
-	@$(CC) $(LOPTS) $(OFILES) $(LIBS) -o dvdisaster
-	@if test -e locale/de/LC_MESSAGES/dvdisaster.mo \
+	$(CC) $(LOPTS) $(OFILES) $(LIBS) -o dvdisaster
+	if test -e locale/de/LC_MESSAGES/dvdisaster.mo \
 	 || echo $(WITH_OPTIONS) | grep "NLS_NO" >/dev/null; \
 	then echo "not touching locale"; \
 	else $(MAKE) --no-print-directory -C locale; \
 	fi
 
 inlined-icons.h: icons/read.png icons/create.png icons/scan.png icons/fix.png icons/verify.png
-	@gdk-pixbuf-csource --raw --name=dvdisaster_read     icons/read.png      >inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_create   icons/create.png   >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_scan     icons/scan.png     >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_fix      icons/fix.png      >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_verify   icons/verify.png   >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_open_ecc icons/open-ecc.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_open_img icons/open-img.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_cd icons/cd.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_gtk_help icons/gtk-help.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_gtk_index icons/gtk-index.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_gtk_preferences icons/gtk-preferences.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_gtk_quit icons/gtk-quit.png >>inlined-icons.h
-	@gdk-pixbuf-csource --raw --name=dvdisaster_gtk_stop icons/gtk-stop.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_read     icons/read.png      >inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_create   icons/create.png   >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_scan     icons/scan.png     >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_fix      icons/fix.png      >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_verify   icons/verify.png   >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_open_ecc icons/open-ecc.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_open_img icons/open-img.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_cd icons/cd.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_gtk_help icons/gtk-help.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_gtk_index icons/gtk-index.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_gtk_preferences icons/gtk-preferences.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_gtk_quit icons/gtk-quit.png >>inlined-icons.h
+	gdk-pixbuf-csource --raw --name=dvdisaster_gtk_stop icons/gtk-stop.png >>inlined-icons.h
 
 $(BUILDTMP)/rs-encoder-sse2.o: rs-encoder-sse2.c
 	@echo "Compiling:" rs-encoder-sse2.c
-	@$(CC) $(SSE2_OPTIONS) $(COPTS) -c rs-encoder-sse2.c -o $(BUILDTMP)/rs-encoder-sse2.o
+	$(CC) $(SSE2_OPTIONS) $(COPTS) -c rs-encoder-sse2.c -o $(BUILDTMP)/rs-encoder-sse2.o
 
 $(BUILDTMP)/rs-encoder-altivec.o: rs-encoder-altivec.c
 	@echo "Compiling:" rs-encoder-altivec.c
-	@$(CC) $(ALTIVEC_OPTIONS) $(COPTS) -c rs-encoder-altivec.c -o $(BUILDTMP)/rs-encoder-altivec.o
+	$(CC) $(ALTIVEC_OPTIONS) $(COPTS) -c rs-encoder-altivec.c -o $(BUILDTMP)/rs-encoder-altivec.o
 
 locale:
-	@$(MAKE) --no-print-directory -C locale
+	$(MAKE) --no-print-directory -C locale
 
 untranslated:
-	@$(MAKE) --no-print-directory -C locale check-untranslated
+	$(MAKE) --no-print-directory -C locale check-untranslated
 
 version.tex:
 	@echo "\\newcommand{\\projectversion}{$(VERSION)}" >$(SRCDIR)/documentation/config/version.tex
@@ -158,11 +158,11 @@ version.tex:
 
 manual: version.tex
 	@echo "Producing user manual... "
-	@$(MAKE) --no-print-directory -C documentation/user-manual manual.pdf
+	$(MAKE) --no-print-directory -C documentation/user-manual manual.pdf
 
 manualclean:
 	@echo "Removing temporary user manual files... "
-	@$(MAKE) --no-print-directory -C documentation/user-manual clean
+	$(MAKE) --no-print-directory -C documentation/user-manual clean
 
 
 # Some usage info
@@ -303,8 +278,8 @@ uninstall:
 
 archclean: distclean
 	@echo "Removing rebuildable documentation parts"
-	@$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/codec-specs archclean
-	@$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/user-manual archclean
+	$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/codec-specs archclean
+	$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/user-manual archclean
 
 distclean: clean
 	@echo "Removing configuration files"
@@ -324,8 +299,8 @@ clean:
 	@find . -name \*.mo -print | xargs rm -f;
 	@find . -name \#\*\# -print | xargs rm -f;
 	@rm -f documentation/config/version.tex
-	@$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/codec-specs clean;
-	@$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/user-manual clean;
+	$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/codec-specs clean;
+	$(MAKE) --no-print-directory -C $(SRCDIR)/documentation/user-manual clean;
 
 BUILD=`grep BUILD $(SRCDIR)/build.h | cut -d ' ' -f 3`
 arch:	archclean time-stamp
